### PR TITLE
lmm investigation

### PR DESCRIPTION
**Pull request by jbloom22**
Using the CI to build as bug only exists in cluster mode.

new OrderedRVD => orderedRVD.apply in LMM.toTable should do run-time ordered-ness check.

### CODE REVIEW

**Comment by jbloom22** [comment]
@tpoterba I think we should go ahead and merge this fix while I continue to investigate what exactly is going on.

File: src/main/scala/is/hail/stats/LinearMixedModel.scala [hunk 1]
```diff
@@ -128,13 +128,17 @@ class LinearMixedModel(hc: HailContext, lmmData: LMMData) {
...
   def fitFullRank(pa_t: RowMatrix): Table = {
     val sc = hc.sc
+    val LMMData(_, nullResidualSq, py0, px0, d0, ydy, xdy0, xdx0, _, _) = lmmData
     val lmmDataBc = sc.broadcast(lmmData)
     val rowTypeBc = sc.broadcast(LinearMixedModel.rowType)
     
     val rdd = pa_t.rows.mapPartitions { itPAt =>
-      val LMMData(_, nullResidualSq, py, px, d, ydy, xdy0, xdx0, _, _) = lmmDataBc.value
+      val py = py0.copy
+      val px = px0.copy
```
Comment (tpoterba) [inline src/main/scala/is/hail/stats/LinearMixedModel.scala]: so you need the extra copies _and_ the serialization instead of broadcast?

Comment (tpoterba) [reply src/main/scala/is/hail/stats/LinearMixedModel.scala]: the extra copies aren't enough by themselves?

**Comment by tpoterba** [comment]
this isn't actually supposed to get merged, right?

**Pull request closed without merging**